packages/Tag/src/Tag.js

```diff
@@ -1,28 +1,38 @@
 import React from "react";
 import PropTypes from "prop-types";
 import RawButton from "@paprika/raw-button";
+import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./Tag.styles";
 import * as types from "./types";
 
 export default function Tag(props) {
   const { children, onRemove, size, onClick, a11yText, avatar, isDisabled, ...moreProps } = props;
+  const I18n = useI18n();
 
   const handleRemove = e => {
     onRemove();
     e.stopPropagation();
   };
 
+  const handleKeyDown = e => {
+    if (e.key === "Enter" || e.key === "Backspace" || e.key === " ") {
```

Comment:
I Included backspace, but open to not including this.

Reply:
This is the only place I'm aware of where the `backspace` key will activate a remove button... but I like it!

But does @sunminsarahlee approve? This means when the keyboard focus is on the little `x` for a selected tag, pressing any of these keys removes it.

Reply:
I would think just "Enter" or "Space" would be the expected buttons to press to action this imo